staff-commands/clean.md

```diff
@@ -33,13 +33,9 @@ This command is used to delete multiple messages from the channel it's used in.
 ### Usage
 
 ```
-m:clean (all|user|matching|notmatching|bots|system|startswith|endswith) [...data] <amount>
+/clean (all|user|matching|notmatching|bots|system|startswith|endswith) [...data] <amount>
```

Comment:
Command usage changed, options are no longer mutually exclusive and can be combined

Reply:
I'm not quite sure how to handle this one. Could you do a Suggested Edit for me?